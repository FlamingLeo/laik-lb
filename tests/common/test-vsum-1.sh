#!/bin/sh
${LAUNCHER-./launcher} -n 1 ../../examples/vsum 100 > test-vsum-1.out
cmp test-vsum-1.out "$(dirname -- "${0}")/test-vsum-1.expected"
