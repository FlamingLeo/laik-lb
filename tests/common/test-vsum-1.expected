Id 0: partitial sums 4950, 4950, 4950, 4950
Total sums: 4950, 4950, 4950, 4950
